deployment/k8s/local-minikube/README.md

```diff
@@ -38,31 +38,77 @@ brew install helm
 helm repo add bitnami https://charts.bitnami.com/bitnami
 ```
 
-# Start on local dev env
-Go to deployment/k8s/local-minikube
-and execute :
-0-mac-install-prerequisites.sh
-1-deploy-customer-os-base-infrastructure-local.sh
-2-build-deploy-customer-os-local-images.sh
-
-## Setup Environment for Ubuntu
+# Setup local environment
+#### 1. Export CUSTOMER_OS_HOME env. variable 
+_(e.g. /Users/alex/workspace/openline-customer-os)_
```

Comment:
No longer needed. The 2 script automatically computes the location of customeros home